examples/platform/openiotsdk/app/openiotsdk_platform.cpp

```diff
@@ -61,6 +62,9 @@ constexpr EndpointId kNetworkCommissioningEndpointSecondary = 0xFFFE;
 
 #define EVENT_TIMEOUT 5000
 
+extern "C" mdh_serial_t * get_example_serial();
+#define SERIAL_BAUDRATE 921600
```

Comment:
Can you add a comment explaining baudrate is too high to be used for sending data into the FVP ?

Reply:
What do you mean? I've just increased the default baud rate to speed up sending data.

Reply:
yes and that makes it impossible to read incoming data into the FVP.

I just want a comment like this.

```
// Consider reducing the baudrate if the serial is used as input and characters are lost  
```